lib/utils/logging.js

```diff
@@ -102,36 +116,40 @@ export function writeErrorLogFile (destination, errorLog) {
     })
 }
 
-export function setupErrorLogging (errorLog) {
+// Init listeners for log messages, transform them into proper format and logs/displays them
```

Comment:
Instead of listening to the same events for displaying, I added another `display` event which get's fired for every log level

But info's are not added to the error log :)